aws/resource_aws_nat_gateway.go

```diff
@@ -38,19 +38,16 @@ func resourceAwsNatGateway() *schema.Resource {
 
 			"network_interface_id": {
 				Type:     schema.TypeString,
-				Optional: true,
 				Computed: true,
 			},
 
 			"private_ip": {
 				Type:     schema.TypeString,
-				Optional: true,
 				Computed: true,
 			},
 
 			"public_ip": {
 				Type:     schema.TypeString,
-				Optional: true,
```

Comment:
should these be moved in the docs too?

Reply:
They are correctly listed only as attributes in the current documentation, unless I'm missing something 😄 

https://www.terraform.io/docs/providers/aws/r/nat_gateway.html